.travis.yml

```diff
@@ -7,9 +7,11 @@ matrix:
     - python: "3.5"
     - python: "3.6"
     # python 3.7 not natively supported yet. https://github.com/travis-ci/travis-ci/issues/9815
-    - python: "3.7"
-      dist: xenial
-      sudo: true
+    # NOTE(patrick): Disabled for now since installing numpy in this xenial / sudo environment
+    # is a headache. Hoping that travis supports python 3.7 soon
```

Comment:
Can we skip numpy altogether in the 3.7 case?

Reply:
we could, but adding that in seemed like a big enough headache that wouldn't be worth it, under the assumption that surely travis would support python 37 soon... right!?

Reply:
haha I would agree, although we've seen first hand how long it can take for external projects to add support for new libraries...